lib/PuppeteerSharp/PageAccessibility/AXNode.cs

```diff
@@ -0,0 +1,255 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+using PuppeteerSharp.Messaging;
+using Newtonsoft.Json.Linq;
+using PuppeteerSharp.Helpers;
+
+namespace PuppeteerSharp.PageAccessibility
+{
+    internal class AXNode
+    {
+        internal AccessibilityGetFullAXTreeResponse.AXTreeNode Payload { get; }
+        public List<AXNode> Children { get; }
+        public bool Focusable { get; set; };
+
+        private readonly string _name;
+        private string _role;
```

Comment:
could this be an enum?

Reply:
It's not clear what's the complete list of roles. We had some headaches in the past because our enums were incomplete.
We could use the official list https://www.w3.org/1999/xhtml/vocab. But we have items like `doc-cover` which are not there.